src/Controller/PublishersController.php

```diff
@@ -104,4 +104,21 @@ public function edit(Request $request, int $id): Response
             'publisher_form' => $form->createView()
         ]);
     }
+
+    /**
+     * @Route ("/deletePublisher/{id}", name="app_deletePublisher")
+     */
+    public function delete(Publisher $publisher): Response
+    {
+        if ($publisher->getGames()->isEmpty()) {
```

Comment:
If you would create a method inside Publisher that would resolve this hasGames() or so, that would be perfect :)